Brewfile.lock.json

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Remove brew lock file and add to .gitignore. Remember you need to commit the delete for the gitignore to work